lib/tasks/ahoy_activity_rollup.rake

```diff
@@ -0,0 +1,79 @@
+namespace :fromthepage do
+  desc "Daily Task to Rollup Ahoy Events into Transcribe minutes by Date, User, and Collection"
+  task ahoy_activity_rollup: :environment do
+    rollup_transcribe_for_date()
+  end
+
+  desc "An ad-hoc task to populate the ahoy rollup for historical data"
+  task ahoy_rollup_crawl: :environment do
+
+    ## Figure out how long we need to count back
+    first_ahoy_event = Date.new(2017, 10, 22).beginning_of_day
+    days = ( (Time.now.beginning_of_day - first_ahoy_event) / 60 / 60 / 24).to_i
+    
+    
+    # Count back from yesterday into the past
+    (1..days).each do |n|
+      date = n.days.ago
+
+      print "\n-- Ahoy Rollup #{date.strftime("%Y-%m-%d")} --\n"
+
+      # Perform the rollup for the day
+      rollup_transcribe_for_date(date)
+    end
+  end
+
+  def rollup_transcribe_for_date(day=1.day.ago)
```

Comment:
I've abstracted the "crawls back to" date to look up the first Ahoy Event in the DB. I've also added a helper rake task that allows us to run this task "For the last N days" in case we miss a day or need to rerun the job. I thought this was better than trying to parse datestring parameters.

This ignores the case for a "date range," but it is easy enough to implement should we need it. I think these two main functions should get us 99% of the way there, though.